### PR TITLE
create catalog

### PR DESCRIPTION
**Pull request by OksanaGrygorchuk**
https://OksanaGrygorchuk.github.io/layout_catalog/
https://OksanaGrygorchuk.github.io/layout_catalog/report/html_report/

### CODE REVIEW

File: src/styles/main.scss [hunk 1]
```diff
@@ -1,3 +1,197 @@
+$main-color: #00acdc;
+$text-color: #060b35;
+$text-second-color: #616070;
```
Comment (nazarmatsevych) [inline src/styles/main.scss:3]: You can move variables to separate file

File: src/styles/main.scss [hunk 2]
```diff
@@ -1,3 +1,197 @@
...
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &--4 &__star:nth-child(-n+4) {
+    background-image: url(../images/star-active.svg);
+  }
```
Comment (nazarmatsevych) [inline src/styles/main.scss:196]: You should use sass for loop here
image

**Review by nazarmatsevych — CHANGES_REQUESTED** [review]
Hey,
very nice and clean work,
pls check my comments  and proofread your code with `[checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)`

**Comment by OksanaGrygorchuk** [comment]
Why did my build test fail? Do I need to change something?

**Review by Viktor-Kost — APPROVED** [review]
Nice!

**Pull request closed without merging**